packages/googlechrome.vm/googlechrome.vm.nuspec

```
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>googlechrome.vm</id>
+    <version>0.0.0.20231210</version>
+    <authors>Mandiant</authors>
+    <description><![CDATA[
+Chrome is a fast, simple, and secure web browser, built for the modern web.
+
+### Notes
+
+- This package uses Chrome's administrative MSI installer and installs the 32-bit on 32-bit OSes and the 64-bit version on 64-bit OSes. If this package is installed on a 64-bit OS and the 32-bit version of Chrome is already installed, the package keeps installing/updating the 32-bit version of Chrome.
+- This package always installs the latest version of Google Chrome, regardless of the version specified in the package. Google does not officially offer older versions of Chrome for download. Because of this you may get checksum mismatch between the time Google releases a new installer, and the package is automatically updated.
+- **If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/issues) if you have problems with the package. Disqus comments will generally not be responded to.**
+]]></description>
```

```suggestion
    <description>Chrome is a fast, simple, and secure web browser, built for the modern web.</description>
```
From [our documentation](https://github.com/mandiant/VM-Packages/wiki/Package-Structure#description):
> The description should be short and not include version specific details or other information that is likely to change in a future version.